e2e/wdio/core/tests/upload-collection.e2e-spec.ts

```diff
@@ -0,0 +1,186 @@
+import { UploadCollectionPo } from '../pages/upload-collection.po';
+import {
+    acceptAlert,
+    click,
+    getAlertText,
+    getAttributeByName,
+    getElementArrayLength,
+    getElementSize,
+    getText,
+    isElementClickable,
+    refreshPage,
+    scrollIntoView,
+    setValue,
+    uploadFile,
+    waitForElDisplayed
+} from '../../driver/wdio';
+import {
+    acceptAlertText,
+    declineAlertText,
+    formatArr,
+    imagePath,
+    imageText,
+    testText
+} from '../fixtures/appData/upload-collection-contents';
+import { checkElArrIsClickable } from '../../helper/assertion-helper';
+
+describe('File uploader component test', () => {
+    const uploadCollectionPage = new UploadCollectionPo();
+    const {
+        uploadCollectionComplexExample,
+        uploadCollectionExample,
+        link,
+        editButton,
+        input,
+        okButton,
+        declineButton,
+        item,
+        uploadCollectionSmallExample,
+        uploadCollectionCustomExample,
+        emphasizedButton,
+        fileUploaderInputFile,
+        checkbox,
+        standardButton
+    } = uploadCollectionPage;
+
+    beforeAll(() => {
+        uploadCollectionPage.open();
+    }, 1);
+
+    beforeEach(() => {
+        refreshPage();
+        waitForElDisplayed(uploadCollectionPage.title);
```

Comment:
for better stability on Safari, please change to 
refreshPage():
waitForPresent(uploadCollectionPage.root)
waitForElDisplayed(uploadCollectionPage.title);